monitor.py

```diff
@@ -226,80 +250,102 @@ def worker(self):
                     ('SystemAddress', self.systemaddress),
                     ('Population', self.systempopulation),
                 ])
+
                 if self.planet:
                     entry['Body'] = self.planet
+
                 entry['Docked'] = bool(self.station)
+
                 if self.station:
                     entry['StationName'] = self.station
                     entry['StationType'] = self.stationtype
                     entry['MarketID'] = self.station_marketid
+
                 self.event_queue.append(json.dumps(entry, separators=(', ', ':')))
+
             else:
-                self.event_queue.append(None)	# Generate null event to update the display (with possibly out-of-date info)
+                # Generate null event to update the display (with possibly out-of-date info)
+                self.event_queue.append(None)
                 self.live = False
 
         # Watchdog thread
-        emitter = self.observed and self.observer._emitter_for_watch[self.observed]	# Note: Uses undocumented attribute
+        emitter = self.observed and self.observer._emitter_for_watch[self.observed]  # Note: Uses undocumented attribute
```

Comment:
I wonder if there's a documented method to achieve this now ?

Reply:
Does not appear to be, went looking

Reply:
Add a comment about self.observer.emitters being a Set, and could check for a member with a 'watch' attribute. But as this is working, leave it be for now, we'll address it if watchdog internals ever change.